rust/src/dhcp/dhcp.rs

```diff
@@ -239,21 +240,21 @@ pub extern "C" fn rs_dhcp_probing_parser(_flow: *const Flow,
 
 #[no_mangle]
 pub extern "C" fn rs_dhcp_tx_get_alstate_progress(_tx: *mut libc::c_void,
-                                                  _direction: libc::uint8_t) -> libc::c_int {
+                                                  _direction: u8) -> libc::c_int {
```

Comment:
nit: The `{` should be on the same line or the next line consistently (IMO). This change is inconsistent with that on line 225.